Aquavolution/Assets/Scripts/playerStats.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class playerStats : MonoBehaviour
+{
+    public int foodCount = 0;
+    private double size = 1;
+    private Vector3 scale = new Vector3(1, 1, 1);
+
+    GameObject player;
+
+    void OnCollisionEnter2D(Collision2D Col)        // This method is called whenever a collision is detected, and passes the Component which Food collided with
+    {
+        Debug.Log("collided with " + Col.collider.ToString().Substring(0, 4));
+        if (Col.collider.ToString().Substring(0, 4) == "Food") {
```

Comment:
instead of Col.collider.ToString().Substring(0, 4) use Col.gameObject.tag ; we want to reduce any hard coded stuff